pkgs/applications/editors/neovim/utils.nix

```diff
@@ -194,12 +194,13 @@ let
         lib.concatStringsSep ";" hostProviderLua;
 
 in
-{
+rec {
   inherit makeNeovimConfig;
   inherit generateProviderRc;
   inherit legacyWrapper;
 
-  buildNeovimPluginFrom2Nix = callPackage ./build-neovim-plugin.nix {
+  buildNeovimPluginFrom2Nix = lib.warn "buildNeovimPluginFrom2Nix was renamed to buildNeovimPlugin" buildNeovimPlugin;
+  buildNeovimPlugin = callPackage ./build-neovim-plugin.nix {
```

Comment:
can we move this into the `let-in` instead so we don't have to use `rec`?

Reply:
I hesitated to so, glad you suggest it :D